customfiles/14_0_INSTALLERCONFIG.sample

```diff
@@ -214,6 +214,9 @@ firstboot_depenguin_run()
 	sleep "${sleep_secs}"
 	for intf in $firstboot_depenguin_interfaces; do
 		echo "$intfs" | grep -Eq "^${intf}$" || continue
+        	if echo "$intf" | grep -Eq "ix"; then
+			ifconfig $intf up
+		fi
```

Comment:
This addresses all comments and turns into into a space-saving oneliner (still readable IMHO)

```suggestion
        	echo "$intf" | grep -Eq "^ix[0-9]$" && ifconfig "$intf" up
```

Reply:
Looks good to me - if we encounter the problem with `ixl` cards in the future (as they are also 10G Intel cards) we can expand the regex. That's why I initially opted for a "wider" approach, but I think you are correct.